src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/VilkårEkspanderbarRad.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { type ReactElement } from 'react';
+
+import styled from 'styled-components';
+
+import { CogIcon, CogRotationIcon, PersonIcon } from '@navikt/aksel-icons';
+import { BodyShort, Table } from '@navikt/ds-react';
+import { RessursStatus } from '@navikt/familie-typer';
+
+import { vilkårFeilmeldingId } from './VilkårTabell';
+import { useBehandling } from '../../../../context/behandlingContext/BehandlingContext';
+import VilkårResultatIkon from '../../../../ikoner/VilkårResultatIkon';
+import type { IVilkårResultat } from '../../../../typer/vilkår';
+import { uiResultat } from '../../../../typer/vilkår';
+import {
+    Datoformat,
+    isoStringTilFormatertString,
+    isoDatoPeriodeTilFormatertString,
+} from '../../../../utils/dato';
+import { alleRegelverk } from '../../../../utils/vilkår';
+
+interface IProps {
+    toggleForm: (visAlert: boolean) => void;
+    vilkårResultat: IVilkårResultat;
+    ekspandertVilkår: boolean;
```

Comment:
rename til erEkspandertVilkår for å tydeliggjøre at det er en boolsk verdi og ikke et faktisk vilkår